contributing.html

```diff
@@ -54,7 +54,28 @@ <h2>Finding A Project To Work On</h2>
 
 		<h2>Becoming a Committer</h2>
 
-		We are always interested in adding new contributors. What we look for is a series of contributions, good taste, and an ongoing interest in the project. If you are interested in becoming a committer, let one of the existing committers know and they can help guide you through the process.
+		We are always interested in adding new contributors. What we look for is a series of contributions, good taste, and an ongoing interest in the project.
+		The following is the guidelines that Kafka PMC is looking for promoting new committers:
```

Comment:
is -> are, but it reads a bit weird. Maybe we should try to rephrase this sentence.

Reply:
ack, I will just say `Kafka PMC looks at the following guidelines for promoting new committers:`